pkg/providers/amifamily/ami.go

```diff
@@ -45,6 +46,7 @@ type Provider interface {
 }
 
 type DefaultProvider struct {
+	sync.RWMutex
```

Comment:
```suggestion
	sync.Mutex
```

This doesn't need to be RW if we're not using the read lock, right? Are there other mutexes that we should just convert to a standard mutex?

Reply:
Added that in. Only found one that should be using the standard mutex